### PR TITLE
[WIP] Updating documentation to remove v2 references and unnecessary steps

### PR DESCRIPTION
**Pull request by ewolinetz**

### CODE REVIEW

**Comment by ewolinetz** [comment]
@rhcarvalho @bparees FYI

**Comment by 0xmichalis** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -30,65 +30,80 @@ Development environment can help you debug problems in your application in the s
 The Perl container is set up so that Apache will load additionally provided .conf files located within the <code>cfg</code> directory of the application's root.  This is useful if you are configuring your application with a database backend and would want to pass through your environment variables to mod_perl with <code>PerlPassEnv</code>.
 
 ###Manual Installation: 
-1. Create an account at [https://www.openshift.com](https://www.openshift.com)  
-*NOTE*: OpenShift Online currently is using V2.
+1. Create an account at [https://www.openshiftapps.com](https://www.openshiftapps.com)
```
Comment (bparees) [inline README.md]: that doesn't go anywhere valid.

File: README.md [hunk 2]
```diff
@@ -30,65 +30,80 @@ Development environment can help you debug problems in your application in the s
 The Perl container is set up so that Apache will load additionally provided .conf files located within the <code>cfg</code> directory of the application's root.  This is useful if you are configuring your application with a database backend and would want to pass through your environment variables to mod_perl with <code>PerlPassEnv</code>.
 
 ###Manual Installation: 
-1. Create an account at [https://www.openshift.com](https://www.openshift.com)  
-*NOTE*: OpenShift Online currently is using V2.
+1. Create an account at [https://www.openshiftapps.com](https://www.openshiftapps.com)  
 2. Fork a copy of [dancer-ex](https://github.com/openshift/dancer-ex)
 3. Clone your repository to your OpenShift server
```
Comment (bparees) [inline README.md]: clone to your development machine, not to the openshift server.

File: README.md [hunk 3]
```diff
@@ -30,65 +30,80 @@ Development environment can help you debug problems in your application in the s
...
 3. Update the GitHub repository url in the instant app configuration to match your forked url
```
Comment (bparees) [inline README.md]: invalid step

File: README.md [hunk 4]
```diff
@@ -30,65 +30,80 @@ Development environment can help you debug problems in your application in the s
...
+1. Create an account at [https://www.openshiftapps.com](https://www.openshiftapps.com)  
 2. Fork a copy of [dancer-ex](https://github.com/openshift/dancer-ex)
 3. Clone your repository to your OpenShift server
 3. Update the GitHub repository url in the instant app configuration to match your forked url 
 2. Add a Perl application from the provided template
-`oc process -f openshift/templates/dancer.json | oc create -f - `
-3. Start the build  
-`oc start-build dancer-app`
+
+		$ oc process -f openshift/templates/dancer.json | oc create -f -
```
Comment (bparees) [inline README.md]: they need to override the source_uri during processing (-v arg) if they're going to use the forked repo.

**Comment by bparees** [comment]
maintaining these instructions in 5 different repos is going to be a pain.  we need to doc in general how to create apps from templates (including overriding the source_uri) in the openshift-docs and point all 5 repos to that doc.

**Comment by rhcarvalho** [comment]
I'm complementing these changes to make all templates minimally match: https://github.com/ewolinetz/dancer-ex/pull/1

**Comment by ewolinetz** [comment]
the changes @rhcarvalho made have been pulled in to my branch and i made the requested changes to the README and updated command output.  i'm going to investigate the docs for referencing regarding template creation (oc process -v ... | oc create ... )

File: README.md [hunk 5]
```diff
@@ -31,66 +31,89 @@ The Perl container is set up so that Apache will load additionally provided .con
 
 ###Manual Installation: 
 1. Create an account at [https://www.openshift.com](https://www.openshift.com)  
-*NOTE*: OpenShift Online currently is using V2.
+  -*NOTE*: OpenShift Online currently is using V2.
```
Comment (bparees) [inline README.md]: just remove this entirely.  

add an intro that just says "You must have an openshift deployment that you have access to in order to deploy this app"

File: README.md [hunk 6]
```diff
@@ -31,66 +31,89 @@ The Perl container is set up so that Apache will load additionally provided .con
...
+		NAME              LABELS                          SELECTOR               IP(S)            PORT(S)
+		dancer-frontend   template=dancer-mysql-example   name=dancer-frontend   172.30.174.142   8080/TCP
+
+In this case, the IP for frontend is 172.30.174.142 and it is on port 8080.  
 *Note*: you can also get this information from the web console.
 
 ###Manual Installation: With MySQL
 1. Create an account at [https://www.openshift.com](https://www.openshift.com)  
-*NOTE*: OpenShift Online currently is using V2.
+  -*NOTE*: OpenShift Online currently is using V2.
```
Comment (bparees) [inline README.md]: again remove and just tell them "we assume you have access to an openshift deployment"

File: README.md [hunk 7]
```diff
@@ -31,66 +31,89 @@ The Perl container is set up so that Apache will load additionally provided .con
...
-                                   dancer-app     172.30.221.178:5000/demo/origin-dancer-app@sha256:dd8452a64d8cf9ba36a461603306ae440b289f819c77dae1989c8980f3243f8e                                                                                                                                       Running      3 minutes    
-mysql-1-4edf0        172.17.0.45                                                                                                                                       ip-10-230-142-143.ec2.internal/10.230.142.143   deployment=mysql-1,deploymentconfig=mysql,name=mysql                                Running      3 minutes   
-                                   mysql          openshift/mysql-55-centos7:latest                                                                                                                                                                                                                        Running      3 minutes  
+
+		$ oc get pods -w
```
Comment (bparees) [inline README.md]: too much repetition in this whole DB section.  it's basically the same steps as the non-DB case, we don't need to document it all twice.

**Comment by ewolinetz** [comment]
Shortened document and inserted statement requiring an OpenShift deployment.  Not sure if we should link to the documentation regarding templates as it will be on the welcome page... thoughts @bparees ?

File: views/default.tt [hunk 8]
```diff
@@ -262,7 +268,7 @@ start;</pre>
                   <p>
                   It will also be necessary to update your application to talk to your database back-end.  The <code>inventory.pm</code> file is configured to use DBI and $ENV in such a way that it will accept environment variables for your connection information that you pass to it.
                   Once an administrator has created a MySQL database service for you to connect with you can add the following environment variables to your deploymentConfig to ensure all your frontend pods have access to these environment variables.
```
Comment (bparees) [inline views/default.tt:270]: "Note: the dancer-mysql.json template creates the DB service and environment variables for you"

**Comment by bparees** [comment]
@ewolinetz one more comment and lgtm.

**Comment by ewolinetz** [comment]
Added note to default.tt.  We should make sure to add a similar note to all other sample-apps that have a template that will create the db for them too

**Pull request merged**